components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLSSOService.java

```diff
@@ -191,4 +201,44 @@ public SAMLSSOReqValidationResponseDTO doSingleLogout(String sessionId)
         return validationResponseDTO;
     }
 
+    /**
+     * Gets all the session participants from session ID send logout requests to them
```

Comment:
add full-stop at the end.

Reply:
fixed with 6b78eeb5f96cfca311630b22611c7896f2e0f096